### PR TITLE
103 update projects to standards

### PR DESCRIPTION
**Pull request by JasonBock**
## Description

Made project updates to target supported .NET frameworks.

### CODE REVIEW

File: RockLib.Threading.Tests/RockLib.Threading.Tests.csproj [hunk 1]
```diff
@@ -1,27 +1,22 @@
...
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-    </PackageReference>
-  </ItemGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\RockLib.Threading\RockLib.Threading.csproj" />
-  </ItemGroup>
-
+	<PropertyGroup>
+		<TargetFrameworks>net6.0;netcoreapp3.1;net48;</TargetFrameworks>
```
Comment (eslutz) [inline RockLib.Threading.Tests/RockLib.Threading.Tests.csproj:3]: Is this line needed here if it's the same as what's specified in the Directory.Build.props?

**Comment by NeilSchiefer** [comment]
Just need to update the README as well as the CHANGLOG with the Unrelease heading and the changes that will be released.  https://github.com/RockLib/RockLib.Reflection.Optimized/pull/6#diff-b6aa1909e64a605db50cfc5ad16226a81448bcafc66f11c733903813a59db341R8

File: RockLib.Threading.Tests/RockLib.Threading.Tests.csproj (hunk 1 above)
Comment (JasonBock) [reply RockLib.Threading.Tests/RockLib.Threading.Tests.csproj:3]: No. This is why we have reviewers ;).

I'll remove it.

**Comment by JasonBock** [comment]
OK, all should be good now...I think.

**Review by NeilSchiefer — APPROVED** [review]

**Comment by NeilSchiefer** [comment]
Woops I missed one thing.  We should wait to bump the version in the main csproj until the publish commit.

**Review by eslutz — APPROVED** [review]
Looks good! 👍🏻

**Review by NeilSchiefer — CHANGES_REQUESTED** [review]
Should change this back to the 1.0.6 until we add the commit to publish so appveyor will publish automatically.

**Comment by JasonBock** [comment]
@NeilSchiefer you mean make the `Version` in the project file `1.0.6`?

**Review by NeilSchiefer — APPROVED** [review]

**Pull request merged**